.pyup.yml

```diff
@@ -0,0 +1,8 @@
+schedule: "every week"
+
+label_prs: enhancement
+
```

Comment:
Finally (that's the last bit, I promise!) would you mind cleaning up these unused blank lines? Not technically necessary, but as they are irrelevant it's better to not have them committed).

Reply:
Okay.

Reply:
I meant the lines from line 4 to the end of the file (the one between `schedule` and `labels_prs` is not required, but sometimes people use it to let it clear that they are talking about a different section of the config file).